mc/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "longname": "Medicines Complete",
+  "name": "mc",
+  "describe": "Recognizes the accesses to the platform Medicines Complete",
+  "contact": "Sean Duffy",
+  "pkb": false,
+  "docurl": "http://analyses.ezpaarse.org/platforms/614248b2f728fb49b59f0783 ",
```

Comment:
Lint: remove irregular whitespace
You can make ``git pull`` in ``platforms`` folder to get an update to trim ``docurl`` value.